src/orion/benchmark/task/profet/forrester.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+from typing import ClassVar, Type
+
+from simple_parsing.helpers.hparams import HyperParameters, uniform
+from logging import getLogger as get_logger
+
+from .profet_task import ProfetTask
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class ForresterTaskHParams(HyperParameters):
+    x: float = uniform(0, 1, discrete=False)
+
+
+class ForresterTask(ProfetTask):
+    """ Simulated Task consisting in training a Random Forrest predictor. """
```

Comment:
Is the Forrester task based on random forest predictors? I thought it was a simple function. I'd need to check again the paper to be sure.

Reply:
I think it's the random forest predictor, but I might be wrong! I'll also check.

Reply:
Indeed you're right, it's a synthetic function. I went ahead and changed the search space of the task to match the description in the paper.
I'm currently unable to download the real data on my laptop, I'll have to double-check that the actual data they used for Forrester (if they even really use it) also match with the space.

Reply:
However, [the code sample here from emukit](https://github.com/EmuKit/emukit/blob/5eeedd30aa7ec14ea2f1e883a12daa0b2d83f0e8/emukit/examples/profet/meta_benchmarks/meta_forrester.py#L57) has `x` as a single hparam in [0,1], instead of the `\alpha` and `\beta` from the paper, so I'm a bit confused! I'll put it back to `x \in [0,1]` for now.

Reply:
From what I understand from the paper, x is the hyperparameter to optimize, and alpha, beta are configurable parameters of the task (not to be optimized, but to define the task itself). They randomly generate 1000 tasks by sampling alpha beta from uniform(0, 1). From the code, they seem to have defined a similar search space for x in uniform(0, 1). 

It seems that they used the forrester task as a synthetic function for experiments described in section 5.2, second paragraph. But they also use PROFET over the forrester task in third paragraph. So it looks there are 2 versions, one synthetic, one simulated.

Reply:
Ok I've created a `Forrester` synthetic function task in `src/orion/benchmark/task/forrester.py`, and changed the hparam types/shapes to match  `x: uniform(0, 1)`.

It's not clear why they would have a meta-model based on a synthetic, easy to run task.. was it to test the "accuracy" of the meta-model?
Should we even include such a task, really?

Reply:
Exactly, they used it to assess the correct behavior of the meta-model:
> The heatmap of statistical comparisons reaches very similar conclusions to those obtained with the original 1000 tasks, contrary to what happened when we did the comparisons with 9 tasks only (i. e. p-values are closer to the original ones). We conclude that our meta-model captures the variability across tasks such that using samples from it (generated based on a subset of tasks) allows us to draw conclusion that are more in line with experiments on the full dataset of tasks than running directly on the subset of tasks.

I think it is worth including it to assess that our wrapper is working properly.

Reply:
Ok so you added it with other tasks, outside of Profet. Makes sense. I think the ForesterTask and other Profet tasts should be renamed something like "{task_name}Profet" so that it makes clear when listing all benchmark tasks than those are tasks based on the Profet meta-model.

Reply:
Fixed in 98dbeaca60c01bdec2b1986134bddd173fded9a8

Reply:
Thanks!